main.go

```diff
@@ -18,8 +19,31 @@ import (
 	"fybrik.io/openmetadata-connector/pkg/utils"
 )
 
+const Customization = "customization"
 const DefaultConfigFile = "/etc/conf/conf.yaml"
 const DefaultCustomizationFile = "./customization.yaml"
+const FailureToParseCustomizationFile = "failure to parse customization file"
```

Comment:
can you put  `DefaultConfigFile` and `DefaultCustomizationFile` as a separate const statement ?
```golang
const (
     DefaultConfigFile = "/etc/conf/conf.yaml"
     DefaultCustomizationFile = "./customization.yaml"
)

const (
   Customization = "customization"
   FailureToParseCustomizationFile = "failure to parse customization file"
   FailureToReadConfigFile = "failure to read configuration file"
)
```